test/e2e/csv_e2e_test.go

```diff
@@ -4171,6 +4174,253 @@ var _ = Describe("ClusterServiceVersion", func() {
 	})
 })
 
+var _ = Describe("Disabling copied CSVs", func() {
+	// Define namespace, operatorGroup, and csv upfront
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: genName("csv-toggle-test-"),
+		},
+	}
+
+	operatorGroup := operatorsv1.OperatorGroup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      genName("csv-toggle-test-"),
+			Namespace: ns.GetName(),
+		},
+	}
+
+	csv := operatorsv1alpha1.ClusterServiceVersion{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      genName("csv-toggle-test-"),
+			Namespace: ns.GetName(),
+		},
+		Spec: operatorsv1alpha1.ClusterServiceVersionSpec{
+			InstallStrategy: newNginxInstallStrategy(genName("csv-toggle-test-"), nil, nil),
+			InstallModes: []operatorsv1alpha1.InstallMode{
+				{
+					Type:      operatorsv1alpha1.InstallModeTypeAllNamespaces,
+					Supported: true,
+				},
+			},
+		},
+	}
+
+	When("an operator is installed in AllNamespace mode", func() {
+		BeforeEach(func() {
+			Eventually(func() error {
+				if err := ctx.Ctx().Client().Create(context.TODO(), ns); err != nil && !k8serrors.IsAlreadyExists(err) {
+					ctx.Ctx().Logf("Unable to create ns: %v", err)
+					return err
+				}
+
+				if err := ctx.Ctx().Client().Create(context.TODO(), &operatorGroup); err != nil && !k8serrors.IsAlreadyExists(err) {
+					ctx.Ctx().Logf("Unable to create og: %v", err)
+					return err
+				}
+
+				if err := ctx.Ctx().Client().Create(context.TODO(), &csv); err != nil && !k8serrors.IsAlreadyExists(err) {
+					ctx.Ctx().Logf("Unable to create csv: %v", err)
+					return err
+				}
+
+				return nil
+			}).Should(Succeed())
+		})
+
+		It("should have Copied CSVs in all other namespaces", func() {
+			Eventually(func() error {
+				requirement, err := k8slabels.NewRequirement(operatorsv1alpha1.CopiedLabelKey, selection.Equals, []string{csv.GetNamespace()})
+				if err != nil {
+					return err
+				}
+
+				var copiedCSVs operatorsv1alpha1.ClusterServiceVersionList
+				err = ctx.Ctx().Client().List(context.TODO(), &copiedCSVs, &client.ListOptions{
+					LabelSelector: k8slabels.NewSelector().Add(*requirement),
+				})
+				if err != nil {
+					return err
+				}
+
+				var namespaces corev1.NamespaceList
+				if err := ctx.Ctx().Client().List(context.TODO(), &namespaces, &client.ListOptions{}); err != nil {
+					return err
+				}
+
+				if len(namespaces.Items)-1 != len(copiedCSVs.Items) {
```

Comment:
nb: Should we check that the namespaces themselves contain the same elements? I believe there is a `gomega` target to do this if it is valuable.